software/gancio.yml

```
@@ -1,10 +1,11 @@
 name: Gancio
 website_url: https://gancio.org/
-description: A shared agenda for local communities.
+description: Local community event and agenda sharing.
```

```suggestion
description: Shared agenda for local communities.
```

I think the previous description was better, don't you think?

---

The old description didn't describe the software in my opinion. 'Shared agenda' wast really clear on its own so I tried to add extra context without changing the description too much. I can revert if smarter people than I understood what it meant! 😆